### ISSUE TITLE
Regex alternate screen garbled

### ISSUE THREAD

**Issue by eugenov** [moemoechu/tabby-highlight#4]
Alternate screens shows garbage symbols for some expressions that use "+" or "*" symbols, i.e.
```
'(.|\r|\n)*?'|"(.|\r|\n)*?"|`(.|\r|\n)*?`
```
(everything in quotes)
or
```
\$\b([_a-zA-Z0-9]+)\b
```
(every word starting with $)
Is there any workaround for this?
Example:
- print with "cat" (regex is working and coloring): [ "$VM" == "" ] || [ "$IP" == "" ] || [ "$NVIDIA" == "" ] && usage
- how it looks inside vim:
[ "__CCCOOONNN_SSSEEEQQQ____CCCOOONNN_SSSEEEQQQ__$VM__CCCOOONNN_SSSEEEQQQ____CCCOOONNN_SSSEEEQQQ__" == "" ] || [ "__CCCOOONNN_SSSEEEQQQ____CCCOOONNN_SSSEEEQQQ__$IP__CCCOOONNN_SSSEEEQQQ____CCCOOONNN_SSSEEEQQQ__" == "" ] || [ "__CCCOOONNN_SSSEEEQQQ____CCCOOONNN_SSSEEEQQQ__$NVIDIA__CCCOOONNN_SSSEEEQQQ____CCCOOONNN_SSSEEEQQQ__" == "" ] && usage

**Comment by moemoechu** [comment]
Just release 1.2.0, found no problem, maybe you can try it.
![image](https://github.com/moemoechu/tabby-highlight/assets/6707029/3bbcaa2a-e1db-45be-9c03-fd8e49eafa24)
![image](https://github.com/moemoechu/tabby-highlight/assets/6707029/20848fae-22a4-44c8-8bab-261f2939f572)

**Comment by moemoechu** [comment]
I found there are some rare situation, the terminal will get mess by placeholder string, I made some fix in next release, maybe version 1.2.4 at tomorrow.

**Comment by moemoechu** [comment]
released 1.2.5,may fix this.

> If you want to elaborate - `#.*` (everything after #) replaced with `[?12l[?25h` with screen redraw when scrolling. I.e. When you open file with `#!/bin/bash` as header, you see `#!/bin/bash`, but if you scroll away and then return to this line - it will look like `[?12l[?25hh` Ah, will leave issue open, if you don't mind. Just to remember... UPD: this happens only when you scroll with mouse wheel. If you scroll with keyboard - everything is fine.
